study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/StudyBuilder.java

```diff
@@ -376,6 +379,20 @@ private void insertStudyDetails(Handle handle, long studyId) {
         }
     }
 
+    private void insertStudySupportedLanguages(Handle handle, long studyId) {
+        String studyGuid = handle.attach(JdbiUmbrellaStudy.class).findById(studyId).getGuid();
+        cfg.getStringList("studySupportedLanguages").forEach(
```

Comment:
Can we make this piece of configuration optional? This means adding
```
if (cfg.hasPath("supportedLanguages")) {
    cfg.getStringList("")...
}

Reply:
Absolutely, I just wasn't sure if this configuration entry is optional. Done.